### PR TITLE
removed hardcoded decimals

### PR DESCRIPTION
**Pull request by talhamalik883**
This PR includes a change to obtain decimals by making an RPC call instead of using a hardcoded value.

### CODE REVIEW

File: backend-node/scripts/erc20Transfer.ts [hunk 1]
```diff
@@ -17,16 +18,17 @@ export const erc20Transfer = async (
...
-  ]);
   // Encode an ERC-20 token transfer to recipient of the specified amount
-  const amountGwei = ethers.utils.parseUnits(amount.toString(), 18); // TODO: if it's USDC on Mumbai and decimals are 6
-  const data = erc20Interface.encodeFunctionData("transfer", [
-    recipientAddress,
-    amountGwei,
-  ]);
+  const readProvider = new ethers.providers.JsonRpcProvider(config.rpcUrl)
+  const tokenContract = new ethers.Contract(tokenAddress, ERC20ABI, readProvider)
+  let decimals = 1
```
Comment (livingrockrises) [inline backend-node/scripts/erc20Transfer.ts:24]: if your try fails decimals would go 1 
make default 18

**Review by livingrockrises — COMMENTED** [review]
done

File: backend-node/scripts/abi.ts [hunk 2]
```diff
@@ -0,0 +1,295 @@
+export const ERC20ABI = [
```
Comment (livingrockrises) [inline backend-node/scripts/abi.ts:1]: actually you need very minimal abi....

**Review by livingrockrises — COMMENTED** [review]
done

File: backend-node/scripts/erc20Transfer.ts (hunk 1 above)
Comment (talhamalik883) [reply backend-node/scripts/erc20Transfer.ts:24]: resolved

File: backend-node/scripts/abi.ts (hunk 2 above)
Comment (talhamalik883) [reply backend-node/scripts/abi.ts:1]: resolved

**Pull request merged**